cmd/remote_client_test.go

```diff
@@ -80,7 +80,10 @@ func TestClient_ShowJobSpec_NotFound(t *testing.T) {
 }
 
 func TestClient_CreateServiceAgreement(t *testing.T) {
-	app, cleanup := cltest.NewApplication()
+	config, _ := cltest.NewConfig()
```

Comment:
We're now doing this regularly. Maybe make a helper for it?

Reply:
@se3000 I think we updated this right as you were reviewing it initially.